landing/src/app/page.tsx

```diff
@@ -0,0 +1,225 @@
+import {DocsFooter, DocsNavBar, DocsTripleCard, DocsHeaderWrapper, DocsHeaderLeft, DocsHeaderRight, DocsHeaderSubtitle, Button, DocsPageLayout, CodeCopy, Card, CardHeader, CardTitle, CardDescription } from '@cqcl/quantinuum-ui'
+import { FaGithub, FaDiscord } from 'react-icons/fa'
+import { LifeBuoyIcon, BookIcon } from "lucide-react";
+
+import Image from 'next/image'
+import Link from 'next/link'
+import { LambeqLogo } from './LambeqLogo';
+
+const navConfig = {
+  navTextLinks: [
+    {
+      title: 'Getting Started',
+      href: '/intro.html',
+    },
+    {
+      title: 'User Guide',
+      href: '/pipeline.html',
+    },
+    {
+      title: 'Tutorials',
+      href: '/tutorials/sentence-input.html',
+    },
+    {
+      title: 'Code Examples',
+      href: '/notebooks.html',
+    },
+    {
+      title: 'API Reference',
+      href: '/root-api.html',
+    },
+  ],
+  navProductName: '\u03BBambeq',
+  navIconLinks: [
+  ],
+}
+
+const footerConfig = {
+  columns: [
+    {
+      title: "Solutions",
+      items: [
+        { name: "Nexus", href: "https://docs.quantinuum.com/nexus" },
+        { name: "TKET", href: "https://docs.quantinuum.com/tket" },
+        { name: "InQuanto", href: "https://docs.quantinuum.com/inquanto" },
+        { name: "\u03BBambeq", href: "https://docs.quantinuum.com/lambeq" },
+      ],
+    },
+    {
+      title: "Hardware",
+      items: [
+        { name: "H-Series", href: "https://docs.quantinuum.com/h-series" },
+        {
+          name: "Get Access",
+          href: "https://www.quantinuum.com/hardware#access",
+        },
+      ],
+    },
+    {
+      title: "Quantinuum",
+      items: [
+        { name: "About", href: " https://www.quantinuum.com/about" },
+        { name: "Research", href: "https://www.quantinuum.com/publications" },
+        { name: "Events", href: "https://www.quantinuum.com/events" },
+      ],
+    },
+  ],
+  subtitle: "",
+};
+
+const cardConfig = [
+  {
+    "title": "Getting Started",
+    "subtitle": "Learn the basics of \u03BBambeq and compositional models of meaning.",
+    icon: "",
+    "image_link": "",
+    "image_description": "",
+    "links": [
+        {
+            "title": "\u03BBambeq pipeline",
+            "description": "Understand how a sentence is converted into a quantum circuit.",
+            "link": "/pipeline.html"
```

Comment:
These links as well.

Reply:
Ah yes, these shouldn't be absolute, don't know why they were